shared/src/commonMain/kotlin/co/touchlab/kampstarter/models/BaseModel.kt

```diff
@@ -2,37 +2,14 @@ package co.touchlab.kampstarter.models
 
 import co.touchlab.kampstarter.printThrowable
 import co.touchlab.kermit.Kermit
-import kotlinx.coroutines.CoroutineExceptionHandler
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.SupervisorJob
 import org.koin.core.KoinComponent
 import org.koin.core.inject
 import org.koin.core.parameter.parametersOf
-import kotlin.coroutines.CoroutineContext
 
 open class BaseModel : KoinComponent {
```

Comment:
Feels like we could just drop this class

Reply:
done